caluma/caluma_form/filters.py

```diff
@@ -64,8 +65,67 @@ class Meta:
         fields = ("meta", "attribute")
 
 
+class VisibleOptionFilter(Filter):
+    """
+    Lookup type display options in the context of a document.
```

Comment:
That sentence reads a bit weird.. maybe rephrase?

```suggestion
    Filter options to only show ones whose `is_hidden` JEXL evaluates to false
```

Reply:
Oh, good catch, I was in the middle of editing this sentence. It was still a mix from when I experimented with the FilterTypes ;)

Fixed.